### PR TITLE
Added information to the 3.11 release notes

### PR DESCRIPTION
**Pull request by ahardin-rh**

### CODE REVIEW

File: release_notes/ocp_3_11_release_notes.adoc [hunk 1]
```diff
@@ -225,6 +256,10 @@ link:https://github.com/openshift/cluster-monitoring-operator[*cluster-monitorin
 As an {product-title} administrator, you can view system resources, containers,
 and components metrics in one dashboard interface, Grafana.
 
+In {product-title} 3.11, the Cluster Monitoring Operator installation is enabled
+by default. Ensure there is an available node in your cluster to avoid
+unexpected failures.
+
```
Comment (jianlinliu) [inline release_notes/ocp_3_11_release_notes.adoc]: Adding more clarification: By default, "Cluster Monitoring Operator" install is expecting 'node-role.kubernetes.io/infra=true' node in your cluster, you also could change it by setting openshift_cluster_monitoring_operator_node_selector in inventory file to your customized node selector.

File: release_notes/ocp_3_11_release_notes.adoc [hunk 2]
```diff
@@ -46,11 +46,85 @@ To upgrade to this release from a previous version, see the
...
-Additionally, the
-xref:../upgrading/blue_green_deployments.adoc#upgrading-blue-green-deployments[blue-green
-installation method] will be deprecated in {product-title} 4.0.
+
+.Features Deprecated in Version 4.0:
+[cols="2",options="header"]
+|====
+|Feature |Justification
+
+|Hawkular
```
Comment (ahardin-rh) [inline release_notes/ocp_3_11_release_notes.adoc]: @sichvoge Please review per https://github.com/openshift/openshift-docs/issues/10458#issuecomment-415665298

@vikram-redhat

File: release_notes/ocp_3_11_release_notes.adoc (hunk 1 above)
Comment (ahardin-rh) [reply release_notes/ocp_3_11_release_notes.adoc]: Updated. Thanks!

**Pull request merged**